browser/components/MarkdownPreview.js

```diff
@@ -491,6 +491,7 @@ export default class MarkdownPreview extends React.Component {
     eventEmitter.on('export:save-md', this.saveAsMdHandler)
     eventEmitter.on('export:save-html', this.saveAsHtmlHandler)
     eventEmitter.on('print', this.printHandler)
+    eventEmitter.on('check-updates', this.checkUpdate)
```

Comment:
I don't see the `checkUpdate` method in this file, is this a mistake? If it not can you explain this to me?

Reply:
Actually pushing the written code would have been a great first step on my end.